wiki/Help_centre/Account_restrictions/en.md

```diff
@@ -81,11 +81,19 @@ Some offences will lead to your account being rolled back upon unrestriction. Th
+- **Your appeal was insufficient or incomplete.** If your appeal does not sufficiently cover the four points listed above, we will request that you write a more complete one. This **does not** necessarily reset the cooldown for your account and you are free to send your new appeal at any point in time! However, if you attempted lying or hiding major rule violations you committed (mainly multi-accounting or cheating), your appeal will be denied for being non-truthful, and your cooldown will be reset to at least **three months**. This is another chance for you to reflect on your actions and make sure you did not forget anything.
+- **You have been evading or playing on other accounts.** Creating another account to be able to play online while your account is restricted is considered an attempt at evading the restriction and will reset your cooldown to at least **three months** from the date you created the account. If you attempt evading too many times, your appeal might be permanently denied. If you are still accessing or playing on your multi-accounts while restricted, your cooldown will also be reset. Remember that multi-accounting is against the [rules](/wiki/Rules) and you are expected to only play on your original account. If you are unsure which account counts as your original account, send us an email and we will let you know!
 - **You continued offending while restricted.** There is no reason to unrestrict someone who continues to break the rules even after having been restricted. If you do continue offending, the cooldown for your account will be reset to your last offence. In the case of extreme or continuous offences, your appeal might be permanently denied.
 - **Your history is too severe.** In cases of extreme offenders, or players who get restricted again after they have already been unrestricted on a successful appeal, we might simply state that your history is too severe to rejoin the game at the current time. Should this apply to you, we recommend waiting **a long time** before appealing again, and making sure you do not break any further rules in that time.
 
+## What happens if this isn't my first time appealing?
+
+In most cases, each restriction on your account doubles the length of the next restriction and the cooldown associated with it.
+
+For example, if you are restricted for cheating, return on appeal, then cheat again, you will be restricted for a minimum of 12 months (double the six months base) before we will review any kind of appeal from you. If you return after this and cheat once again, you will be restricted for a minimum of 24 months.
```

Comment:
since it says "minimum" here too, is 24 not the cap?

Reply:
min terminology is used here so as not to infer that it's a "you're restricted for 24 months then you get to come back" when it's "you're restricted for 24 months THEN you can apply to remove that restriction", the latter causing restriction to last longer than 24 months

Reply:
oh right, not cooldowns, nvm